client/components/language-picker/language-names.js

```diff
@@ -0,0 +1,141 @@
+	'he': translate => translate( 'Hebrew' ),
+	'hi': translate => translate( 'Hindi' ),
+	'hr': translate => translate( 'Croatian' ),
+	'hu': translate => translate( 'Hungarian' ),
+	'hy': translate => translate( 'Armenian' ),
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
+	'ii': translate => translate( 'Sichuan Yi' ),
+	'ilo': translate => translate( 'Iloko' ),
+	'is': translate => translate( 'Icelandic' ),
+	'it': translate => translate( 'Italian' ),
+	'ja': translate => translate( 'Japanese' ),
+	'ka': translate => translate( 'Georgian' ),
+	'kir': translate => translate( 'Kyrgyz' ),
+	'kk': translate => translate( 'Kazakh' ),
+	'km': translate => translate( 'Khmer' ),
+	'kn': translate => translate( 'Kannada' ),
+	'ko': translate => translate( 'Korean' ),
+	'ks': translate => translate( 'Kashmiri' ),
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
+	'lo': translate => translate( 'Lao' ),
+	'lt': translate => translate( 'Lithuanian' ),
+	'lv': translate => translate( 'Latvian' ),
+	'mk': translate => translate( 'Macedonian' ),
+	'ml': translate => translate( 'Malayalam' ),
+	'mn': translate => translate( 'Mongolian' ),
+	'mr': translate => translate( 'Marathi' ),
+	'ms': translate => translate( 'Malay' ),
+	'mt': translate => translate( 'Maltese' ),
+	'mwl': translate => translate( 'Mirandese' ),
+	'nah': translate => translate( 'Nahuatl' ),
+	'nap': translate => translate( 'Neapolitan' ),
+	'nds': translate => translate( 'Low German' ),
+	'ne': translate => translate( 'Nepali' ),
+	'nl': translate => translate( 'Dutch' ),
+	'nn': translate => translate( 'Norwegian Nynorsk' ),
+	'no': translate => translate( 'Norwegian' ),
+	'non': translate => translate( 'Old Norse' ),
+	'nv': translate => translate( 'Navajo' ),
+	'oci': translate => translate( 'Occitan' ),
+	'or': translate => translate( 'Odia' ),
+	'os': translate => translate( 'Ossetic' ),
+	'pa': translate => translate( 'Punjabi' ),
+	'pl': translate => translate( 'Polish' ),
+	'ps': translate => translate( 'Pashto' ),
+	'pt': translate => translate( 'Portuguese' ),
+	'pt-br': translate => translate( 'Brazilian Portuguese' ),
+	'qu': translate => translate( 'Quechua' ),
+	'ro': translate => translate( 'Romanian' ),
+	'ru': translate => translate( 'Russian' ),
+	'rup': translate => translate( 'Aromanian' ),
+	'sc': translate => translate( 'Sardinian' ),
+	'si': translate => translate( 'Sinhala' ),
+	'sk': translate => translate( 'Slovak' ),
+	'sl': translate => translate( 'Slovenian' ),
+	'snd': translate => translate( 'Sindhi' ),
+	'so': translate => translate( 'Somali' ),
+	'sq': translate => translate( 'Albanian' ),
+	'sr': translate => translate( 'Serbian' ),
+	'su': translate => translate( 'Sundanese' ),
+	'sv': translate => translate( 'Swedish' ),
+	'ta': translate => translate( 'Tamil' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Tamil Nadu' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).